kasa/iot/device.py

```diff
@@ -115,7 +116,7 @@ class SmartDevice:
 
     Examples:
         >>> import asyncio
-        >>> dev = SmartDevice("127.0.0.1")
+        >>> dev = Device("127.0.0.1")
```

Comment:
This won't work anymore without importing it in a proper way. Maybe the preferred way should be documented?